### ISSUE TITLE
make prod-yaml doesn't work on Linux

### ISSUE THREAD

**Issue by andrewsykim** [kubernetes-sigs/cluster-api-provider-vsphere#294]
On Linux, the output of `kustomize version` is 
```
$ kustomize version
Version: {KustomizeVersion:2.0.3 GitCommit:a6f65144121d1955266b0cd836ce954c04122dc8 BuildDate:2019-03-05T20:37:42Z GoOs:linux GoArch:amd64}
```

Notice `2.0.3` not `v2.0.3, this breaks the Makefile and some scripts in a few places because it assumes the semantic version has `v2` which seems to only be true on darwin.

**Comment by akutz** [comment]
Hi @andrewsykim,

There's a fix for this in PR #283.

**Comment by andrewsykim** [comment]
Ah, will check there thanks
